docs/reference2/sql-syntax/lexical-structure.md

```diff
@@ -75,6 +189,8 @@ Not a function/operator/identifier, have meta-meaning for the commands.
 
 TODO: does ksqlDB have multi-line comments?
 
+bracketed comments
+'/*' .*? '*/' 
```

Comment:
For bit below on operator precedence...

Precedence in order:

1.  * (Multiplication), / (Division), % (Modulus)
2. + (Positive), - (Negative), + (Addition), + (Concatenation), - (Subtraction)
3. =, >, <, >=, <=, <>, != (Comparison operators)
4. NOT
5. AND
6. BETWEEN, LIKE, OR

When two operators in an expression have the same precedence level, they're evaluated left to right based on their position in the expression.

And of course you can use brackets to force or make precedence clear, e.g. `(5 + 2) * 3`